pkgs/by-name/sh/shadershark/update.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+data="$(curl "$repo_url/rss-log")"
+
+rev="$(echo "$data" | grep 'rev/' | head -n 1 | sed 's;^.*rev/\([^<]*\)[^$]*;\1;')"
+rawTimestamp="$(echo "$data" | grep pubDate | head -n 1 | sed 's;^.*<pubDate>\([^<]*\)[^$]*;\1;')"
+
+new_version="$(date +%Y-%m-%d -d "$rawTimestamp")"
```

Comment:
The date should be of the commit

Reply:
It is from the commit. The date matches the latest commit date.

Reply:
Upstream just started using tags